cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -74,7 +75,15 @@ func GetMetricStorageFactory(name string, host component.Host) (storage.MetricSt
 	return mf, nil
 }
 
-func GetStorageFactoryV2(name string, host component.Host) (tracestore.Factory, error) {
+func GetTraceStoreFactory(name string, host component.Host) (tracestore.Factory, error) {
```

Comment:
do we still have use for `GetStorageFactory`? Would be nice to make it private or remove

Reply:
@yurishkuro yeah - i left a comment about this at https://github.com/jaegertracing/jaeger/issues/5079#issuecomment-2525257234